doctr/models/detection/zoo.py

```diff
@@ -11,8 +11,8 @@
 __all__ = ["detection_predictor"]
 
 default_cfgs: Dict[str, Dict[str, Any]] = {
-    'db_resnet50': {'model': 'db_resnet50', 'post_processor': 'DBPostProcessor'},
-    'linknet': {'model': 'linknet', 'post_processor': 'LinkNetPostProcessor'},
+    'db_resnet50': {'model': 'db_resnet50'},
+    'linknet': {'model': 'linknet'},
 }
```

Comment:
We can remove this now

Reply:
and replace L25 by `_model = detection.__dict__[arch](pretrained=pretrained)`

Reply:
mmmh, you mean that it could be a subset of what's available as detection architectures?